content/learning-paths/11111111-1111-1111-1111-111111111111/mastering-kubernetes-for-engineers/introduction-to-devops-and-site-reliability-enginerring/ci-cd/quiz.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Quiz"
+passing_percentage: 70
+questions:
+    - id: "q1"
+      text: "What is Continuous Integration (CI)?"
+      type: "single-answer"
+      marks: 2
```

Comment:
@Vincamine, we default to two points per question, correct? `marks` can be omitted, yes?

@Namanv0509 @vr-varad @zihanKuang please confirm and be aware.